frontend/src/pages/main.js

```diff
@@ -1,5 +1,31 @@
-import React from "react";
+import React from 'react';
+import Nav from '../components/global/Nav';
```

Comment:
It gives me a warning that Nav is unused

Reply:
Nav has been removed